src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.spec.ts

```diff
@@ -21,35 +22,17 @@ describe('TelemetryActivationNotificationComponent', () => {
   let fixture: ComponentFixture<TelemetryNotificationComponent>;
 
   let authStorageService: AuthStorageService;
-  let userService: UserService;
```

Comment:
The assignment ```userService = TestBed.get(UserService);``` in line 70 needs to be removed as well (https://github.com/ceph/ceph/pull/37043/files#diff-e75f5a013eaa445efcbe83fdf9b8fe8dL69)